gecko.py

```diff
@@ -114,6 +114,15 @@ def price(coin):
 def query_coin(coin):
     return get("coins", coin)
 
+def query_coins_markets(coin):
```

Comment:
This should also pass in `coins`. We are passing in a list. You could equivalently pass in `*coin`.